src/views/Account.vue

```diff
@@ -32,13 +32,13 @@
           </button>
         </div>
         <div class="account-container_actions">
-          <router-link :to="`/accounts/${accountId}/${asset}/send`"><button class="account-container_actions_button">
+          <router-link :to="`/accounts/${accountId}/${asset}/send`"><button class="account-container_actions_button" id="send">
             <div class="account-container_actions_button_wrapper"><SendIcon class="account-container_actions_button_icon" /></div>Send
           </button></router-link>
-          <router-link :to="`/accounts/${accountId}/${asset}/swap`"><button class="account-container_actions_button">
+          <router-link :to="`/accounts/${accountId}/${asset}/swap`"><button class="account-container_actions_button" id="swap">
```

Comment:
Should things like "swap" should be more specific incase it conflicts with other IDs in the app?

Reply:
This should work on the wallet asset screen, but the idea to add an assert name to it so that it will be unique, will do that in next